tests/Doctrine/Tests/Persistence/ObjectManagerDecoratorTest.php

```diff
@@ -22,7 +22,7 @@ public function __construct(ObjectManager $wrapped)
 
 class ObjectManagerDecoratorTest extends TestCase
 {
-    /** @var MockObject|ObjectManager */
+    /** @var MockObject&ObjectManager */
```

Comment:
Not sure how IDE will react to that. Maybe in an @psalm-var?

Reply:
Interesting, I've been using it for a while because `PHPStorm` understands it, don't know others.

Reply:
IIRC, PHPStorm basically treat that the same way as "|" now. There was an issue on their bug tracker where they talked about that and they were not sure of what to do with that